Makefile

```diff
@@ -24,7 +24,7 @@ CLUSTERCTL_URL ?= https://github.com/kubernetes-sigs/cluster-api/releases/downlo
 SONOBUOY_VERSION ?= 0.19.0
 SONOBUOY_URL ?= https://github.com/heptio/sonobuoy/releases/download/v$(SONOBUOY_VERSION)/sonobuoy_$(SONOBUOY_VERSION)_$(OPERATING_SYSTEM)_amd64.tar.gz
 TESTPKGS ?= github.com/talos-systems/talos/...
-RELEASES ?= v0.7.1 v0.8.0
+RELEASES ?= v0.7.1 v0.8.2
```

Comment:
testing now with 0.8.3-pre-release which has a fix for control plane checkpoint annotations

Reply:
In fact, I can do 0.8.3 now